docs/bnb_greenfield/guide/introduction/introduction.md

```diff
@@ -0,0 +1,116 @@
+---
+
+## What is BNB Greenfield
+
+BNB Greenfield is a cutting-edge decentralized storage and blockchain storage solution, designed to harness the power of decentralized technology in the field of data ownership and the data economy. The platform focuses on providing decentralized data management and access, to revolutionize the data economy by simplifying the process of storing and managing data, while connecting data ownership with the DeFi context of BNB Chain.
+
+What sets Greenfield apart from existing centralized and decentralized storage systems are its three key components:
+
+- It allows Ethereum-compatible addresses to create and manage both data and token assets.
+
+- It natively links data permissions and management logic onto BSC as exchangeable assets and smart contract programs with all other assets.
+
+- It provides developers with similar API primitives and performance as popular existing Web2 cloud storage.
+
+The ultimate goal of Greenfield is to establish a foundation for new data economy and dApp models, which will undoubtedly aid in the development and evolution of the foundation for Web3.
+
+### How BNB Greenfield Works
+
+BNB Greenfield operates on two layers:
+
+1. It is built on a new, storage-focused blockchain; and
+
+2. It consists of a network of "storage providers".
+
+The [BNB Greenfield Blockchain](../greenfield-blockchain/overview.md) maintains the ledger for users and records the storage metadata as the common blockchain state data. Its native token for gas and governance is BNB, which is transferred from BNB Smart Chain. Additionally, BNB Greenfield blockchain has its own staking logic designed for governance.
+
+The [Storage Providers (SP)](../storage-provider/introduction/overview.md) are storage service infrastructures provided by organizations or individuals that use Greenfield as the ledger and the single source of truth. Each SP is responsible for responding to user requests to upload and download data, while also serving as the gatekeeper for user rights and authentications.
+
+Together, BNB Greenfield blockchain and the SPs comprise a decentralized object storage system that serves as the core of this new economy. Developers can construct decentralized applications (dApps) using the BNB Greenfield platform, which can act as client tools that facilitate user interactions with Greenfield; or applications that provide significant value to users' real lives using Greenfield as their infrastructure. These applications will use blockchain addresses as user identifiers and connect with features and smart contracts on the Greenfield blockchain, Greenfield SPs, and BNBChain.
+
+A native cross-chain bridge exists between BSC and BNB Greenfield blockchain. While it is cheaper to create and access data on Greenfield, the relevant data operation can be transferred to BSC and integrated with DeFi smart contract systems to generate new business models.
+
+### Explore BNB Greenfield
```

Comment:
Can we combine `Explore BNB Greenfield` and `How to Participate in the Ecosystem` to `Explore and Participate in BNB Greenfield`, and put at the bottom of this page.

Reply:
yes. Done